RockLib.DistributedTracing.AspNetCore/AspNetCore/CorrelationIdMiddleware.cs

```diff
@@ -5,49 +5,49 @@
 
 namespace RockLib.DistributedTracing.AspNetCore
 {
-    /// <summary>
-    /// A middleware used to set the correlation id in the response header. The id will either be found
-    /// in the request header, or if not found, generated.
-    /// </summary>
-    public class CorrelationIdMiddleware
-    {
-        private readonly RequestDelegate _next;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="CorrelationIdMiddleware"/> class.
-        /// </summary>
-        /// <param name="next">The <see cref="RequestDelegate"/> used to process the HTTP request.</param>
-        /// <param name="options">The options for the <see cref="CorrelationIdMiddleware"/> instance.</param>
-        public CorrelationIdMiddleware(RequestDelegate next, IOptionsMonitor<CorrelationIdMiddlewareOptions> options = null)
-        {
-            _next = next ?? throw new ArgumentNullException(nameof(next));
-
-            var middlewareOptions = options?.CurrentValue ?? new CorrelationIdMiddlewareOptions();
-
-            HeaderName = middlewareOptions.HeaderName ?? HeaderNames.CorrelationId;
-        }
-
-        /// <summary>
-        /// Gets the name for the correlation id header.
-        /// </summary>
-        public string HeaderName { get; }
-
-        /// <summary>
-        /// Sets the correlation id header to the response, if missing.
-        /// </summary>
-        /// <param name="context">The HttpContext.</param>
-        public async Task InvokeAsync(HttpContext context)
-        {
-            var correlationId = context.GetCorrelationId(HeaderName);
-
-            context.Response.OnStarting(() =>
-            {
-                if (string.IsNullOrWhiteSpace(context.Response.Headers[HeaderName]))
-                    context.Response.Headers[HeaderName] = correlationId;
-                return Task.CompletedTask;
-            });
-
-            await _next(context).ConfigureAwait(false);
-        }
-    }
+   /// <summary>
```

Comment:
Should we be going to 4 spaces if it wasn't?

Reply:
I don't know if we've specified that in `.editorconfig`...I just checked, and it isn't. We have `indent_style = space`...do we want `indent_size = 4` or `indent_size = 3`? I don't care, either way is fine (and I'll also make an update to our assets file as well).

Reply:
I have a habit of just hitting Ctrl+E+D, which formats a document.

What's our current tab style? 4 spaces? 3 spaces?